common/reviews/api/rush-lib.api.md

```diff
@@ -734,6 +734,18 @@ export interface IStopwatchResult {
     toString(): string;
 }
 
+// @beta
+export interface ISubspaceConfigurationJson {
+    // Warning: (ae-forgotten-export) The symbol "ISubspaceConfig" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    availableSubspaces: ISubspaceConfig[];
```

Comment:
Available?

Reply:
In the RFC we called it `"subspaceNames"`:
```js
{
  "useSubspaces": true,

  // Names must be lowercase and separated by dashes.
  // To avoid mistakes, common/config/subspaces/ subfolder
  // cannot be used unless its name appears in this array.
  "subspaceNames": [ "default", "react19", "install-test" ]
}
```

I'm not sure "available" is even correct -- adding a string to this list will probably cause Rush to read and validate its config files whether or not any projects are actually using that subspace.

Reply:
@william2958 Could you update your PR description to list any design changes you made that are different from the RFC doc?

Reply:
Changed to match the RFC spec